### PR TITLE
complete canSave methods

### PR DESCRIPTION
**Pull request by nyckoo**

### CODE REVIEW

File: LogisticApp/ViewModel/Forms/PersonFormViewModel.cs [hunk 1]
```diff
@@ -68,8 +68,42 @@ public override void loadData(BaseEntity entity)
         //TODO add validation if there will be enough time
         public override void save()
         {
-            _addressViewModel.save();
-            Creator.createOrUpdate("person", this.serializeData());
+            if (canSave())
```
Comment (mtyton) [inline LogisticApp/ViewModel/Forms/PersonFormViewModel.cs:71]: This is not exactly proper to check canSave here, we'll discuss this tommorow

File: LogisticApp/ViewModel/Forms/PersonFormViewModel.cs [hunk 2]
```diff
@@ -68,8 +68,42 @@ public override void loadData(BaseEntity entity)
...
+            {
+                _addressViewModel.save();
+                Creator.createOrUpdate("person", this.serializeData());
+            }
+            else
+                Console.WriteLine("Empty textbok or incorrect data.");
```
Comment (mtyton) [inline LogisticApp/ViewModel/Forms/PersonFormViewModel.cs:77]: using WriteLine in window app is not the best idea 😄

**Review by mtyton — COMMENTED** [review]
This works but It does not look like like it should 😢

File: LogisticApp/ViewModel/Forms/PersonFormViewModel.cs (hunk 2 above)
Comment (nyckoo) [reply LogisticApp/ViewModel/Forms/PersonFormViewModel.cs:77]: yup, wasn't sure how to use this method in context of a program and wanted you to check this out :)

**Pull request closed without merging**